src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseService.java

```diff
@@ -27,7 +27,13 @@
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
-import de.tum.in.www1.artemis.domain.*;
```

Comment:
Feel free to ignore the teamscale warning. We decided for us that star imports are ok for 4+ import from the same package.